### ISSUE TITLE
Will you release training code?

### ISSUE THREAD

**Issue by StayYouth1993** [syguan96/DynaBOA#18]
How many dataset have you used?
Can you share some training experience to achieve the performance as paper showed？ex：
1. used pretrained model or train from zero
2. how many loss used and loss weight
3. opt method
4. training dataset and weight
5. training epoch

Best！

**Comment by syguan96** [comment]
Hi, I only trained the basemodel with Human 3.6M and its SMPL annotation (named mosh). The model is trained from scratch. The used losses and their weight are the same as SPIN, but I removed the optimization module since Human 3.6M already contains SMPL annotation. To accelerate the training, I used apex and DDP. I remember that I trained about 90 epochs with 3x2080ti.

The training code is dirty now. You can re-train the model by yourself first. If I have time, I will reorganize the code and release it.

**Comment by StayYouth1993** [comment]
Ok, thanks a lot! I'll try later!

**Comment by funnyQZQ** [comment]
Hi @syguan96 , any update for training code?
